libs/tup-components/src/tickets/TicketsTable.tsx

```diff
@@ -1,11 +1,12 @@
 import React, { useMemo } from 'react';
 import { Column, Row } from 'react-table';
+import { Link } from 'react-router-dom';
 import {
   InfiniteScrollTable,
   InlineMessage,
   LoadingSpinner,
 } from '@tacc/core-components';
-import { Ticket, useTickets } from '@tacc/tup-hooks';
+import { Ticket, useGetTickets } from '@tacc/tup-hooks';
 import { DateCreated, Status, Subject } from './TicketsCells';
 import './TicketsView.scss';
```

Comment:
Requests:

1. Rename to `TicketsView.global.css`. [Reference.](https://github.com/TACC/tup-ui/blob/task/tup-ui-styles/apps/tup-ui/src/styles/README.md)
2. Convert SCSS syntax—if any—to native CSS or  [PostCSS syntax](https://github.com/csstools/postcss-plugins/tree/main/plugin-packs/postcss-preset-env#glossary).

<sub>I know you didn't even add that stylesheet in this PR, but if you're doing it for the other ones, please do this one too if you have bandwidth.</sub>

Reply:
I renamed `TicketsView.scss` to `TicketsTable.global.css`. This way it's more apparent which component file the CSS file is being imported into -- `TicketsTable.global.css` is being imported into `TicketsTable.tsx`. S